traffic_portal/app/src/app.js

```diff
@@ -425,13 +425,13 @@ var trafficPortal = angular.module('trafficPortal', [
 
             RestangularProvider.setBaseUrl(ENV.api['root']);
 
-            RestangularProvider.setResponseInterceptor(function(data, operation, what) {
+            RestangularProvider.setResponseInterceptor(function(data, operation) {
 
-                if (angular.isDefined(data.response)) { // todo: this should not be needed. need better solution.
-                    if (operation == 'getList') {
-                        return data.response;
+                if (angular.isDefined(data.response)) {
```

Comment:
not sure how i feel about this change. this is making a very global change to add a simple redirect for one use case as i understand it. this would require a lot of testing of TP.

Reply:
i'll give it some more thought.